src/main/java/com/jfrog/GradleDependencyTreeUtils.java

```diff
@@ -19,6 +20,8 @@
  * @author yahavi
  **/
 public class GradleDependencyTreeUtils {
+    // The maximum number of times a single dependency is populated, during the run of each configuration.
+    private static final int MAX_DEP_POPULATIONS_IN_CONFIG = 10;
```

Comment:
Let's use the term "appearances" instead of "populations" in all places
```suggestion
    private static final int MAX_DEP_APPEARENCES_IN_CONFIG = 10;
```

Reply:
This number doesn't represent the number of appearances of a dependency in the graph, but the number of times a dependency is populated. Because the graph is flat, a dependency can be populated 10 times (the maximum allowed), but it'll have many more appearances in the graph.